content/docs/setup/kubernetes/sidecar-injection.md

```diff
@@ -135,7 +135,7 @@ with the option `sidecarInjectorWebhook.enabled` set to `false`. E.g.
 ```command
 $ helm template --namespace=istio-system --set sidecarInjectorWebhook.enabled=false install/kubernetes/helm/istio > istio.yaml
 $ kubectl create ns istio-system
-$ kubectl apply -f istio.yaml
+$ kubectl apply -n istio-system -f istio.yaml
```

Comment:
I think we do not need to specify `namespace` here as we already specified `namespace` when creating the template.

Reply:
Let's keep it - there is a bit of confusion about where it goes, this makes it a bit more consistent.